WooCommerce/WooCommerceUITests/Flows/MockDataReader.swift

```diff
@@ -39,8 +39,14 @@ class GetMocks {
         return updatedData
     }
 
-    static func readNewProductData() throws -> ProductData {
-        let originalData = try JSONDecoder().decode(NewProductMock.self, from: self.getMockData(test: ProductsTests.self, filename: "products_add_new"))
+    static func readNewProductData(productType: String) throws -> ProductData {
+        let file = [
+            "physical": "products_add_new_physical_2129",
+            "virtual": "products_add_new_virtual_2123",
+            "variable": "products_add_new_variable_2131"
+        ]
+
```

Comment:
Swift nitpick: This dictionary is all constants. If we extracted it as a `private static let` class property, it would be instantiated only once rather than every time this method is called.

I don't think it makes much difference from the point of view of the memory footprint, but I think it would make for "better looking" code.

Reply:
updated in https://github.com/woocommerce/woocommerce-ios/pull/9441/commits/be54c931c0b6857b9d3681ae6226d2ddf96b05d2